lib/assessment.js

```diff
@@ -8,15 +8,24 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(lastResult, thisNum) {
+  if (!lastResult) {
```

Comment:
This is an interesting approach. I don't see why it shouldn't work with a few changes.

Here's an interesting case to consider. Suppose `min` is invoked with arguments `0` and some positive number, e.g., `min(0, 111)`. Zero is a falsey number, so this first `if` condition succeeds and `min` returns the second argument, `111`.

If you're testing for `lastResult` being undefined or null, compare it directly to them in the `if` condition: `if(lastResult === null || lastResult === undefined)`. 3/4 credit.